### ISSUE TITLE
Trainable variable bug in BatchNormalization layer with tensorflow backend

### ISSUE THREAD

**Issue by LynnHo** [fchollet/keras#6818]
I use BatchNormalization layer with the tensorflow backend, and I found a issue that when i call tf.trainable_variables(), the moving_mean and the moving_variance appear in the trainable variables.

Is it a bug?
